nats-base-client/jsutil.ts

```diff
@@ -94,6 +94,11 @@ export function checkJsErrorCode(
   }
   description = description.toLowerCase();
   switch (code) {
+    case 408:
+      return NatsError.errorForCode(
+        ErrorCode.JetStream404NoMessages,
```

Comment:
A 408 is not a "no message", it is a request timeout or no longer valid, but not a "no message"

Reply:
the server is sending me that in response...